src/aliceVision/imageMasking/imageMasking.cpp

```diff
@@ -0,0 +1,113 @@
+// This file is part of the AliceVision project.
+// Copyright (c) 2019 AliceVision contributors.
+// This Source Code Form is subject to the terms of the Mozilla Public License,
+// v. 2.0. If a copy of the MPL was not distributed with this file,
+// You can obtain one at https://mozilla.org/MPL/2.0/.
+
+#include "aliceVision/imageMasking/imageMasking.hpp"
+#include "aliceVision/imageMasking/eigen2cvHelpers.hpp"
+
+#include "aliceVision/image/Image.hpp"
+#include "aliceVision/image/io.hpp"
+
+#include <opencv2/opencv.hpp>
+#include <opencv2/core/core.hpp>
+#include <opencv2/core/eigen.hpp>
+#include <opencv2/imgproc.hpp>
+
+namespace aliceVision{
+namespace imageMasking{
+
+namespace{
+  float clamp(float value)
```

Comment:
I did indent anonymous namespace to make obvious that included functions are local. I feel it's useful and does not hurt readability too much because only few functions are defined in anonymous namespace.

Reply:
for completely new file I'd prefer to stick with the new formatting rules, we need to start at some point. But it\s true that we need to do that in batch at some point.